lib/ts/utils/windowHandler/index.ts

```diff
@@ -12,38 +12,45 @@
  * License for the specific language governing permissions and limitations
  * under the License.
  */
-import { defaultWindowHandlerImplementation } from "./defaultImplementation";
-import { WindowHandlerInterface, WindowHandlerInput } from "./types";
+import { defaultWindowHandlerImplementation } from './defaultImplementation'
+import { WindowHandlerInterface, WindowHandlerInput } from './types'
 
 export default class WindowHandlerReference {
-    private static instance?: WindowHandlerReference;
-
-    windowHandler: WindowHandlerInterface;
+    private static instance?: WindowHandlerReference
+
+    // Declare subset of WindowHandlerReference that is actually used,
+    // so implementors don't need to over-engineer.
+    windowHandler: {
+        location: Pick<
```

Comment:
We shouldn't limit this to a subset. This lib exports the window handler, and the other props are used by supertokens-web-js and supertokens-auth-react.

Reply:
Fair, I shouldn't have committed this